### PR TITLE
feature/migration-app-fixes: Fix past transgressions

### PR DESCRIPTION
**Pull request by AltusBarry**

### CODE REVIEW

File: authentication_service/user_migration/views.py [hunk 1]
```diff
@@ -209,7 +211,7 @@ def get_success_url(self, query=None):
         return redirect(url)
 
     def get_login_url(self, query=None):
```
Comment (cobusc) [inline authentication_service/user_migration/views.py]: `query` argument is never used.

File: authentication_service/user_migration/views.py [hunk 2]
```diff
@@ -234,12 +236,13 @@ def dispatch(self, *args, **kwargs):
...
                 salt="ge-migration-user-pwd-gate-passed",
-                max_age=5*360 # 5 min in seconds
+                max_age=5*360  # 5 min in seconds
             )
         except signing.SignatureExpired:
             messages.error(
                 self.request,
-                _("Password reset url has expired, please login again.")
+                _("Password reset url has expired,"
+                    " please restart the password reset proces.")
```
Comment (cobusc) [inline authentication_service/user_migration/views.py]: Odd indent

File: authentication_service/user_migration/views.py [hunk 3]
```diff
@@ -267,3 +270,6 @@ def migration_user(self):
 
     def get_success_url(self, query=None):
         return redirect(reverse("password_reset_done"))
+
+    def get_login_url(self, query=None):
```
Comment (cobusc) [inline authentication_service/user_migration/views.py]: `query` argument never used.

**Comment by cobusc** [comment]
:+1: Just check if some of those unused arguments cannot be removed.

File: authentication_service/user_migration/views.py (hunk 1 above)
Comment (AltusBarry) [reply authentication_service/user_migration/views.py]: Added them in, preparation for an issue I know we will be dealing with. This flow does not drop the user back on login with the next parameter still present. The entire pwd reset flow doesn't cater for that, not just migration user pwd reset.

File: authentication_service/user_migration/views.py (hunk 2 above)
Comment (AltusBarry) [reply authentication_service/user_migration/views.py]: I blindly let flake8 tell me what to do today! Glad it looks odd to someone else too. Will revert rather.

**Pull request closed without merging**